packages/config/config/devices/0x021c/ilock15.json

```diff
@@ -5,8 +5,8 @@
 	"description": "Smart Door Lock",
 	"devices": [
 		{
-			"productType": "0x5010",
-			"productId": "0x1001",
+			"productType": "0x634b",
+			"productId": "0x504c",
 			"zwaveAllianceId": 3048
```

Comment:
https://products.z-wavealliance.org/products/3048 lists the product ID you moved to the new file, along with a picture of a lock. Not sure what's correct, but this doesn't seem to be long here (anymore).

Reply:
Also done :)